.github/workflows/release.yml

```diff
@@ -1,52 +1,90 @@
 name: Release
 
-# Uncomment the following to let goreleaser automatically
-# create a GitHub release when a tag is pushed.
-# permissions:
-#   contents: write
+env:
+  APPLICATION: "op-erigon"
+  BUILDER_IMAGE: "ghcr.io/goreleaser/goreleaser-cross:v1.21.13"
+  DOCKER_BASE_IMAGE: "alpine:3.20.2"
+  APP_REPO: "testinprod/op-erigon"
+  PACKAGE: "github.com/testinprod-io/op-erigon"
+  DOCKERHUB_REPOSITORY: "testinprod/op-erigon"
+  DOCKERFILE_PATH: "./Dockerfile.release"
+  #GITHUB_AUTOMATION_EMAIL: "github-automation@erigon.tech"
+  #GITHUB_AUTOMATION_NAME: "Erigon Github Automation"
+  LABEL_DESCRIPTION: "Erigon is an implementation of Ethereum (execution layer with embeddable consensus layer), on the efficiency frontier. Archive Node by default."
 
 on:
   push:
     branches-ignore:
       - '**'
-    tags:
-      - 'v*.*.*'
-      # to be used by fork patch-releases ^^
-      - 'v*.*.*-*'
+    #branches:
+    #  - 'master'
+    #tags:
+      ## only trigger on release tags:
+      #- 'v*.*.*'
+      #- 'v*.*.*-*'
```

Comment:
Note that release workflow now works on manual workflow trigger(`build-release`) instead of automatically running when new tag is created. This change is introduced in upstream.